### PR TITLE
Spinkube support policy

### LINKED ISSUE #250: Document support policy for SpinKube
It would be helpful to include either a footer or excerpt on the [Troubleshooting guide ](https://www.spinkube.dev/docs/contrib/troubleshooting/) about SpinKube's support policy. This will help set expectation with users, and can be repurposed as the support reference on SpinKube marketplace offerings as well. I'm imagining the message would be something along the lines of "SpinKube provides support on best effort basis - please file an issue under the appropriate repository and if urgent, amplify in Slack"

### PR DESCRIPTION
**Pull request by macolso**
This PR adds a section to the troubleshooting guide for SpinKube that covers the support policy. Closes #250

### CODE REVIEW

File: content/en/docs/contrib/troubleshooting.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ aliases:
 The following is a list of common error messages and potential troubleshooting suggestions that
 might assist you with your work.
 
+## SpinKube Support Policy
+
+SpinKube provides support on best effort basis. Please file an issue under the appropriate sub-project repository([Spin Operator](https://github.com/spinkube/spin-operator/issues/new), [`spin kube` plugin](https://github.com/spinkube/spin-plugin-kube/issues/new), [Runtime Class Manager](https://github.com/spinkube/runtime-class-manager/issues/new), [Containerd Spin Shim](https://github.com/spinkube/containerd-shim-spin/issues/new)) and if urgent, amplify in [Slack](https://cloud-native.slack.com/archives/C06PC7JA1EE).
```
Comment (calebschoepp) [inline content/en/docs/contrib/troubleshooting.md:14]:
```suggestion
SpinKube provides support on a best effort basis. Please file an issue under the appropriate sub-project repository([Spin Operator](https://github.com/spinkube/spin-operator/issues/new), [`spin kube` plugin](https://github.com/spinkube/spin-plugin-kube/issues/new), [Runtime Class Manager](https://github.com/spinkube/runtime-class-manager/issues/new), [Containerd Spin Shim](https://github.com/spinkube/containerd-shim-spin/issues/new)) and if urgent, amplify in [Slack](https://cloud-native.slack.com/archives/C06PC7JA1EE).
```

**Review by calebschoepp — APPROVED** [review]
LGTM!

File: content/en/docs/contrib/troubleshooting.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (macolso) [reply content/en/docs/contrib/troubleshooting.md:14]: > I would not expect users would know exactly what repos to ask the question / raise the issue without knowing proper debugging instruments.
That's a good point! I'd like to still have a CTA for users experience issues who didn't come through the marketplace. What do you think about sending users who didn't come through Azure to the Spin Operator repo to start and redirecting from there?

[the remark this answers is not in the document]
Comment (Mossaka) [reply content/en/docs/contrib/troubleshooting.md:14]: yea, it makes sense to me

**Pull request merged**